models/models.js

```diff
@@ -72,8 +68,32 @@ const fetchAllArticles = (sort_by='created_at',  order='DESC') => {
          })
 }
 
+const fetchCommentsByArticleId = (article_id) => {
+    if(isNaN(article_id)){
+        return Promise.reject({status: 400, msg:'Bad request'})
+    }
+    return db
+    .query(`SELECT 
+            comments.comment_id,
+            comments.votes,
+            comments.created_at,
+            comments.author,
+            comments.body,
+            comments.article_id
+            FROM comments
+            WHERE comments.article_id = $1
+            ORDER BY created_at DESC;`, [article_id])
+    .then((body) => {
+        if(body.rows.length === 0){
```

Comment:
if there are no comments returned, are we sure that this means the article id doesnt exist? what about articles that havent been commented on yet?